apps/web/src/components/TimeLine/Desktop/TimeHeader.tsx

```
@@ -0,0 +1,24 @@
+  return (
+    <ul
+      className="pb-19 flex items-center"
+      style={{ width: `${timeSlots.length * 72}px` }}
+    >
+      {timeSlots.map((slot) => (
+        <TimeText
+          isHalfHour={slot.isHalfHour}
+          isCurrentTime={false}
+          isTestCurrentTime={slot.time === testCurrentTime}
+          time={!slot.isHalfHour ? slot.time : undefined}
+        />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)